flink-staging/flink-streaming/flink-streaming-core/src/main/java/org/apache/flink/streaming/api/functions/source/SocketTextStreamFunction.java

```diff
@@ -69,14 +68,14 @@ public void run(SourceContext<String> ctx) throws Exception {
 
 	public void streamFromSocket(SourceContext<String> ctx, Socket socket) throws Exception {
```

Comment:
I think this method should be private because it is not meant to be used outside this class.